CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.0] - 2024-02-05
+### Added
+- Support for PHP 8.2
```

Comment:
This is not correct. It should say PHP 8.3.

Reply:
thank you for reporting!
it is fixed: https://github.com/karriereat/pdf-merge/pull/26